IGSIOCalibration/vtkIGSIOLandmarkDetectionAlgo.cxx

```diff
@@ -321,7 +321,7 @@ int vtkIGSIOLandmarkDetectionAlgo::GetNearExistingLandmarkId( double stylusTipPo
     landmarkDifference_Reference[1] = detectedLandmark_Reference[1] - stylusTipPosition_Reference[1];
     landmarkDifference_Reference[2] = detectedLandmark_Reference[2] - stylusTipPosition_Reference[2];
     landmarkDifference_Reference[3] = detectedLandmark_Reference[3] - stylusTipPosition_Reference[3];
-    if( vtkMath::Norm( landmarkDifference_Reference ) < this->MinimunDistanceBetweenLandmarksMm / 3 )
+    if( vtkMath::Norm( landmarkDifference_Reference ) < this->MinimumDistanceBetweenLandmarksMm / 3 )
```

Comment:
Comment on why divide by 3

Reply:
It may be because if the distance between landmarks is X then if we are X/3 distance from a landmark then we can be quite confident that we are actually at that landmark (other landmarks should be farther, at about X*2/3 distance). If we used X/2 as distance threshold then we may accept a point that is halfway between two landmarks and we would not want that. You can add this as a comment.